assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -17,109 +15,218 @@ import {getAvailableParameters} from './selectors/parameter-data';
+                select(`#expansion-container-row-${parameter.parameterCode}`)
+                    .attr('hidden', select(`#expansion-container-row-${parameter.parameterCode}`).attr('hidden') !== null ? null : 'true');
+
+                // If the icon is open, close it and vice versa
+                // Note - there are two open/close icons for each parameter, but only one will show at a time at
+                // any given screen size. Both icons need to be synced.
+                ROW_TOGGLE_ICON_TYPES.forEach(typeOfIcon => {
+                    const rowToggle = select(`#expansion-toggle-${typeOfIcon}-${parameter.parameterCode}`);
+                    rowToggle.attr('aria-expanded') === 'true' ?
+                        rowToggle.attr('class',  ROW_TOGGLE_CLOSED_CLASS) :
+                        rowToggle.attr('class', ROW_TOGGLE_OPENED_CLASS);
+                    rowToggle.attr('aria-expanded') === 'true' ?
+                        rowToggle.attr('aria-expanded', 'false') :
+                        rowToggle.attr('aria-expanded', 'true');
+                });
+            });
     }
+};
 
-    const columnHeaders = ['   ', 'Parameter', 'Period of Record', 'WaterAlert'];
-    const tableContainer = container.append('div')
-        .attr('id', 'select-time-series');
-
-    tableContainer.append('label')
-        .attr('id', 'select-time-series-label')
-        .text('Select a time series');
-    const table = tableContainer.append('table')
-        .classed('usa-table', true)
-        .classed('usa-table--borderless', true)
-        .attr('aria-labelledby', 'select-time-series-label')
-        .attr('tabindex', 0)
-        .attr('role', 'listbox');
-
-    table.append('thead')
-        .append('tr')
-            .selectAll('th')
-            .data(columnHeaders)
-            .enter().append('th')
-                .attr('scope', 'col')
-                .text(d => d);
-    table.append('tbody')
-        .attr('class', 'usa-fieldset')
-        .selectAll('tr')
-        .data(Object.values(parameters))
-        .enter().append('tr')
-        .attr('id', d => `time-series-select-table-row-${d.parameterCode}`)
+/*
+* Helper function that draws the main containing rows. Note the 'parameter selection' is a nested USWD grid. This section of code
+* creates the main grid rows for each parameter and adds on click functions. Later, another grid will be nested in these rows.
+* @param {Object} Store - The application Redux state
+* @param {String} siteno - A unique identifier for the monitoring location
+* @param {Object} element - The target element on which to append the row
+* @param {Object} parameter - Contains details about the current parameter code
+*/
+const drawContainingRow = function(store, siteno, element, parameter) {
+    return element.append('div')
+        .attr('id', `container-row-${parameter.parameterCode}`)
+        .attr('class', 'grid-container grid-row-container-row')
```

Comment:
You can use USWDS mixins to get the "grid" styling. I wonder if that might make things a bit simpler. Look here https://designsystem.digital.gov/utilities/layout-grid/#sass-mixins and scroll down to the Mixins section

Reply:
Makes sense. Done.